quay-enterprise/prometheus.md

```diff
@@ -1,6 +1,6 @@
 # Prometheus metrics under Quay Enterprise
 
-Quay Enterprise exports a [Prometheus](prometheus.io)-compatible endpoint on each instance to allow for easy monitoring and alerting.
+Quay Enterprise exports a [Prometheus][prometheus.io]-compatible endpoint on each instance to allow for easy monitoring and alerting.
```

Comment:
Because `prometheus.io` is defined at the bottom of the document, so it's a reference link.